e2e-tests/desktop/pages/sessionPage.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import GlobalPage from './globalPage.js';
+
+export default class SessionPage {
+  constructor(page) {
+    this.page = page;
+  }
+
+  async cancelAllSessions() {
+    while (
+      (await this.page
+        .getByRole('row')
+        .filter({ hasNot: this.page.getByRole('columnheader') })
+        .getByLabel('Cancel')
```

Comment:
I think this is good to be specific in the table as I totally forgot about the other `Canceling` text on the screen, but I wonder if we should have just done `getByLabel('Cancel', {exact: true})` instead? Maybe we should do it now either way?

Reply:
Yeah sorry I should have been clearer, I like the filter but I do think it's better to have the `exact`